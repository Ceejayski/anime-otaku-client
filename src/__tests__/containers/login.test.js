import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Login from '../../containers/login';
import store from '../../redux';

test('renders Correctly', () => {
  const Placeholder = () => (
    <Provider store={store}>
      <BrowserRouter>
        <Login />
      </BrowserRouter>
    </Provider>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
