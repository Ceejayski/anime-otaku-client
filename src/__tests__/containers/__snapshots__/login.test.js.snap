// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
<div
  className="background-overlay container"
>
  <div
    className="d-flex justify-content-center align-items-center w-100 mb-auto"
    style={
      Object {
        "height": "100vh",
        "marginTop": "-40px",
      }
    }
  >
    <div
      className=" border-0 w-100"
      style={
        Object {
          "maxWidth": "300px",
        }
      }
    >
      <h4
        className="text-center"
      >
        Login
      </h4>
      <p
        className="text-center"
      >
        Hey there, Sign in and we take it up from there
      </p>
      <form
        className="w-100 mb-5"
        onSubmit={[Function]}
      >
        <div
          className="form-group mb-4"
        >
          <div>
            <input
              className="form-control"
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="Email"
              style={
                Object {
                  "borderRadius": "25px",
                }
              }
              type="email"
              value=""
            />
          </div>
        </div>
        <div
          className="form-group mb-4"
        >
          <div>
            <input
              className="form-control"
              name="password"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="Password"
              style={
                Object {
                  "borderRadius": "25px",
                }
              }
              type="password"
              value=""
            />
          </div>
        </div>
        <div
          className="form-group text-center"
        >
          <button
            className="btn btn-primary btn-block btn-lg border-0"
            disabled={false}
            style={
              Object {
                "backgroundColor": "#FF5617",
                "borderRadius": "25px",
              }
            }
            type="submit"
          >
            <span>
              Login
            </span>
          </button>
        </div>
        <button
          disabled={true}
          style={
            Object {
              "display": "none",
            }
          }
        />
      </form>
      <p
        className="text-center"
      >
         
        Register new Account?
         
        <a
          href="/signup"
          onClick={[Function]}
        >
          Create Account
        </a>
      </p>
    </div>
  </div>
</div>
`;
